### ISSUE TITLE
Phase 2 SFT Stage 1 Execution (Foundation)

### ISSUE THREAD

**Issue by nochadisfaction** [pixelatedempathy/pixelated#548]
# Phase 2 Stage 1 SFT Execution: Therapeutic Foundation

## Pre-requisites

* H100 GPU Instance provisioned and stable.
* S3 `pixel-data` canonical source natively mounted out or configured via Environment Secrets for our stream loader script.

## Execution Procedure

1. Execute the Stage 1 hyperparameter config over the canonical corpus `s3://pixel-data/final_dataset/shards/curriculum/stage1/`.

```bash
uv run python ai/lightning/production/train_therapeutic_ai.py --stage 1
```

2. Confirm Tensor/Loss monitoring within PyTorch Lightning loop initializes successfully over bfloat16.
3. Once running, establish systemd monitor or wrap the execution within a detached `tmux` daemon to handle network interruption via SSH proxy.
4. Export the resulting base fine-tune weights to `s3://pixel-data/models/wayfarer_12b_stage_1_foundation`.
